ui/src/app/tasks/task-executions/task-executions.component.html

```diff
@@ -0,0 +1,34 @@
+<h1 [ngBusy]="busy">Task Executions</h1>
+<p>
+  This section lists all the available task executions.
+</p>
+
+<div *ngIf="!taskExecutions">No task executions available.</div>
```

Comment:
I suppose it's better to keep it as otherwise page would be empty in case it fails.